py/desispec/scripts/procexp.py

```diff
@@ -30,7 +24,7 @@ def parse(options=None):
     parser.add_argument('--sky', type = str, default = None,
                         help = 'path of DESI sky fits file')
     parser.add_argument('--calib', type = str, default = None,
-                        help = 'path of DESI calibration fits file')
+                        help = 'path of DESI calibration fits file')xs
```

Comment:
typo that prevents this script from even being imported; i.e. this branch needs more testing.